lib/demo_web/router.ex

```diff
@@ -1,12 +1,15 @@
 defmodule DemoWeb.Router do
   use DemoWeb, :router
 
+  import DemoWeb.UserAuth
+
   pipeline :browser do
     plug :accepts, ["html"]
     plug :fetch_session
     plug :fetch_flash
     plug :protect_from_forgery
     plug :put_secure_browser_headers
+    plug :authenticate_user
```

Comment:
We will probably need to ask the users to add this line manually in the post-generation instructions.